libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationViewModel.kt

```diff
@@ -21,25 +26,45 @@ internal class DropInNavigationViewModel : ViewModel() {
     private val _navigationState = MutableStateFlow<NavigationState>(NavigationState.FreeDrive)
     val navigationState = _navigationState.asStateFlow()
 
+    @Suppress("PropertyName")
+    val _activeNavigationStarted = MutableStateFlow(false)
+    val activeNavigationStarted = _activeNavigationStarted.asStateFlow()
```

Comment:
The _activeNavigationStarted and _navigationState make sense as part of the NavigationStateViewModel

Reply:
They already have been moved to RoutesState. I forgot to remove these.